### DISCUSSION TITLE
PERFORMERS – I want to audition for theater companies which produce content I find meaningful

### DISCUSSION THREAD

**Discussion by ybressler** [pair-cg/general-contributions#35] [User Stories]
# 1. Need:
I want to work for a theater company which produce content I find meaningful.
> _As an example, I want to work for productions which focus on environmental issues._

# 2. Action:
Search for the opportunity. And apply.

> **Details:** Search against current open roles and filter for productions with or about:
> * environmental issues
> * role criteria (age / gender / ethnicity)
> * location of the performance
> * pay?
> * union / non-union
> _(There are definitely additional criteria, but this is the gist of it.)_

# 3. Desired Outcome:
* I found enough opportunities that I can apply to.

Including other stakeholders:
* I'm offered a role.

# 4. Perspective:
Actors / performers looking for their next gig.

# 5. Stakeholder Group:
* Performers
* Casting Directors
* Arts Organizations
* Search Engines
* Data platforms
  * IMBDB
  * IBDB
* Professional networking platforms
  * backstage
  * playbill
  * broadway world

# 6. Additional info:
Actors dedicate a _ton_ of time applying for prospective work. It's arduous work.

**Comment by samipierre** [comment]
Hi @ybressler 

Thanks for your user story. I have some follow ups. 

1. Need: you want to work for these companies
2. Action: you search for opportunities, you apply for these opportunities 
      a.   what search engines (google, Facebook, bing, etc.) are you using to look for rolls?
3. Desire outcome: get a roll, invited for audition
___
# Original
> # 1. Need:
> As an actor, I want to apply to theater companies which produce content I find meaningful. As an example, I want to audition for productions which focus on environmental issues.
> 
> # 2. Action:
> Search against current open roles and filter for productions with or about:
> 
> * environmental issues
> * role criteria (age / gender / ethnicity)
> * location of the performance
> * pay?
> 
> _(There are definitely additional criteria, but this is the gist of it.)_
> 
> # 3. Desired Outcome:
> I get to see all productions which have open roles which fit my criteria.
> 
> # 4. Perspective:
> Actors / performers looking for their next gig.
> 
> # 5. Stakeholder Group:
> * Performers
> * Casting Directors
> * Arts Organizations
> * Search Engines
> * Data platforms
>   
>   * IMBDB
>   * IBDB
> * Professional networking platforms
>   
>   * backstage
>   * playbill
>   * broadway world
> 
> # 6. Additional info:
> Actors dedicate a _ton_ of time applying for prospective work. It's arduous work.

___
# Edited

> > "As an actor, I want to apply to theater companies which produce content I find meaningful. As an example, I want to audition for productions which focus on environmental issues."
> 
> ## Need:
> --> Correction: I want to **work for a theater company which produce content I find meaningful.**
> 
> ## Action:
> Search for the opportunity. And apply.
> 
> ## Desired Outcome:
> > "I get to see all productions which have open roles which fit my criteria."
> 
> * I found enough opportunities that I can apply to.
>   Including other stakeholders:
> * I'm offered a role.

**Reply by ybressler** [reply]
Made my changes.
